src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Literal.cs

```diff
@@ -16,28 +18,25 @@ internal partial class LocalRewriter
     {
         public override BoundNode VisitLiteral(BoundLiteral node)
         {
-            return MakeLiteral(node.Syntax, node.ConstantValue, node.Type, oldNodeOpt: node);
+            return MakeLiteral(node.Syntax, node.ConstantValue!, node.Type, oldNodeOpt: node);
```

Comment:
assert?

Reply:
I'm not sure how this is safe, `BoundLiteral.ContantValue` is annotated?

Reply:
It is because we are in lowering (we wouldn't lower if a constant value was an error and thereby had no constant value).  But I'll add an assertion.

---
In reply to: [390516983](https://github.com/dotnet/roslyn/pull/42068#discussion_r390516983) [](ancestors = 390516983)